packages/editor/src/editor/canvas/drag-data.ts

```diff
@@ -37,12 +43,21 @@ export const isDropZoneDisabled = (dropId: string, dropType?: ComponentType, act
     dragId === dropId ||
     `${STRUCTURE_DROPZONE_ID_PREFIX}${dragId}` === dropId ||
     `${TABLE_DROPZONE_ID_PREFIX}${dragId}` === dropId ||
+    `${COLUMN_DROPZONE_ID_PREFIX}${dragId}` === dropId ||
     dragId === preDropId
   ) {
     return true;
   }
 
   const data = active.data.current;
+
+  if (
+    (dropId.startsWith(COLUMN_DROPZONE_ID_PREFIX) || parentDropId?.startsWith('datatablecolumn')) &&
```

Comment:
What gives you the guarantee that this id always starts with 'datatablecolumn'?
Only use types for such checks! The id is not stable and typesafe!